listenbrainz/webserver/views/playlist_api.py

```diff
@@ -1023,23 +1034,31 @@ def export_playlist_jspf(service):
     """
     user = validate_auth_header()
 
-    if service != "spotify":
+    if service != "spotify" and service != "apple_music":
         raise APIBadRequest(f"Service {service} is not supported. We currently only support 'spotify'.")
 
-    spotify_service = SpotifyService()
-    token = spotify_service.get_user(user["id"], refresh=True)
+    if service == "spotify":
+        spotify_service = SpotifyService()
+        token = spotify_service.get_user(user["id"], refresh=True)
+    elif service == "soundcloud":
+        apple_service = AppleService()
```

Comment:
Doesn't look right. Should it be this?
```suggestion
    elif service == "apple_music":
        apple_service = AppleService()
```